### DISCUSSION TITLE
[JPA] 기본적인 어노테이션

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#112] [1. 개발 관련 글 모음]
JPA 어렵네용...
[게시글 링크](https://amaran-th.github.io/Spring/[JPA]%20%EA%B8%B0%EB%B3%B8%EC%A0%81%EC%9D%B8%20%EC%96%B4%EB%85%B8%ED%85%8C%EC%9D%B4%EC%85%98%20%EC%A0%95%EB%A6%AC/)

**Comment by shkisme** [comment]
이제 JDBC로 돌아갈 수 없으실 겁니다 껄껄...
